rollup.config.js

```diff
@@ -47,6 +51,36 @@ const getUMDConfig = ({ env, file }) => {
 };
 
 export default [
-  getUMDConfig({ env: 'development', file: 'dist/react-beautiful-dnd.js' }),
+  getUMDConfig({ env: 'development', file: 'dist/react-beautiful-dnd.umd.js' }),
+
   getUMDConfig({ env: 'production', file: 'dist/react-beautiful-dnd.min.js' }),
+
+  {
+    input,
+    output: {
+      file: pkg.main,
+      format: 'cjs',
+    },
+    external: isExternal,
+    plugins: [
+      resolve({ extensions }),
+      babel(getBabelOptions()),
+      strip({ debugger: true }),
+    ],
+  },
+
+  {
+    input,
+    output: {
+      file: pkg.module,
+      format: 'es',
+    },
+    external: isExternal,
+    plugins: [
+      resolve({ extensions }),
+      babel(getBabelOptions()),
+      strip({ debugger: true }),
```

Comment:
I am keen to keep console.* statements for the non-minified builds. Can the `strip` function be updated so that only the minified build strips console.*? https://github.com/rollup/rollup-plugin-strip#usage

Reply:
From non minified umd too? It seems console was always stripped from both umd bundles